src/lib/components/consignments/components/artwork_consignment_todo.tsx

```diff
@@ -86,45 +86,58 @@ const DoneButton = () =>
     <Image source={require("../images/black-tick.png")} />
   </ImageBG>
 
+const Button = (props: TouchableHighlightProperties) =>
```

Comment:
If you set the type of the stateless component the following way

```javascript
const Button: React.SFC<TouchableHighlightProperties> =
 ```

`props.children` will be defined